frontend/src/components/TranslucentModalDiv.tsx

```diff
@@ -0,0 +1,11 @@
+import React, {ReactNode} from 'react';
```

Comment:
eslint gives me a warning for unused-vars

Reply:
Good catch!